src/vsr/superblock.zig

```diff
@@ -958,6 +970,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
             vsr_state.commit_max = update.commit_max;
```

Comment:
(A few lines above this) we can assert that `update.sync_view >= update.checkpoint.header.view`.